src/axom/spin/BVH.hpp

```diff
@@ -230,11 +253,53 @@ class BVH
    * \pre y != nullptr if dimension==2 || dimension==3
    * \pre z != nullptr if dimension==3
    */
-  void find( IndexType* offsets, IndexType* counts,
-             IndexType*& candidates, IndexType numPts,
-             const FloatType* x,
-             const FloatType* y,
-             const FloatType* z=nullptr ) const;
+  void findPoints( IndexType* offsets, IndexType* counts,
```

Comment:
Suggestion:  Would ``findCandidatesForPoints()`` and ``findCandidatesForRays()`` be clearer?